coreppl/src/coreppl-to-mexpr/compile.mc

```diff
@@ -140,7 +140,8 @@ lang MExprCompile =
         match loadCompiler options method with (_, compile) in
         match mapLookup method runtimes.entries with Some entry then
           let ast = transformModelAst options ast in
-          let ast = compileModel compile entry id model in
+          let ast = compileModel compile entry id {model with ast = ast} in
+         -- let ast = compileModel compile entry id model in
```

Comment:
@gizemcaylak Can you remove the commented out line? No point in leaving it in the code